service/routes.py

```diff
@@ -26,12 +26,25 @@ def index():
     """ Root URL response """
     return ("<p>The inventory service keeps track of how many of each product we have in our warehouse</p>",
             status.HTTP_200_OK)
-
+    
 
 ######################################################################
 #  R E S T   A P I   E N D P O I N T S
 ######################################################################
 
+######################################################################
+# DELETE A INVENTORY ITEM
+######################################################################
+@app.route("/inventory/<int:product_id>/<string:condition>", methods=["DELETE"])
+def delete_inventory(product_id, condition):
+    '''This endpoint will delete a product with the specified id and condition'''
+    app.logger.info("Request to delete a product with product_id %s and condition %s", product_id, condition)
+    product = Inventory.query.filter_by(product_id=product_id, condition=condition).first()
```

Comment:
Just wondering as to why Inventory.find(...) isn't going to work here?

Reply:
Yes, both are same, internally find does the same. Changed to find() for code uniformity.